lib/config/options/index.ts

```diff
@@ -2003,7 +2003,7 @@ const options: RenovateOptions[] = [
     description:
       'Text added here will be placed last in the PR body, with a divider separator before it.',
     type: 'string',
-    default: `This PR has been generated by [Renovate Bot](https://github.com/renovatebot/renovate).`,
+    default: `This PR has been generated by [Renovate](https://github.com/renovatebot/renovate).`,
```

Comment:
My change is breaking a test, but I updated the Jest snapshots already. I don't know what I should do to fix the failing tests. Can a maintainer help me?